### PR TITLE
Initial changes to updating/upgrading to latest patternfly

### PR DESCRIPTION
**Pull request by carlosmmatos**
Upgrade process for using the latest version of Patternfly. I think there are few more things we will need to hash - but at least here it will be visible to more than just me.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,13 +1,26 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.31.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```
Comment (isimluk) [inline go.sum:3]: Please ditch `go.sum` changes from this PR.

Let's keep them separated as they may cause different kind of issues.

**Comment by carlosmmatos** [comment]
I'll get rid of go.sum

**Comment by carlosmmatos** [comment]
@isimluk Let's keep this in draft for this quarter - I will make this a priority for Q4. I'll start working on the content changes/updates to the current site for this quarter.

**Comment by carlosmmatos** [comment]
@isimluk rebased on top of master - and added fix to navigation.tsx that was preventing the site from fully loading. Apparently between the React.Fragment and inline conditional operator for adding the NavGroup title, it wasn't too happy about it. Not sure if this is a react upgrade issue or not. Going to keep investigating other updates

**Pull request closed without merging**